File: asml/LC26RemoveDuplicatesFromSortedArray.py
class Solution(object):
    def removeDuplicates(self, nums):
        if not nums or len(nums) == 1:
            return
        start = 0
        for i in range(1, len(nums)):
            if nums[i] != nums[start]:
                nums[start+1] = nums[i]
                start += 1
        return start +1

File: asml/LC26RemoveDuplicatesFromSortedArray.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if (nums.size() == 0) return 0;
        int start = 0;
        for (int i = 1; i < nums.size(); i++) {
            if (nums[i] != nums[start]) {
                nums[start+1] = nums[i];
                start++;
            }
        }
        return start+1;
    }
};
